lib/process-services/task-list/components/task-filters.component.ts

```diff
@@ -155,8 +158,14 @@ export class TaskFiltersComponent implements OnInit, OnChanges {
      * @param filter
      */
     public selectFilter(filter: FilterRepresentationModel) {
-        this.currentFilter = filter;
-        this.filterClick.emit(filter);
+        if (filter) {
+            this.currentFilter = this.filters.find(obj =>
```

Comment:
can we use a better name for this variable instead of obj? I don't e.g. currentFilter